bootstrap.js

```diff
@@ -0,0 +1,2 @@
+require('babel-core/register');
```

Comment:
instead of transpiling code on the fly can we babel it to lib directory and then deploy. lib directory should be in .gitignore list